### DISCUSSION TITLE
Environments only showing latest commit

### DISCUSSION THREAD

**Discussion by newtewt** [community/community#37391] [Actions]
We create a release branch for deploying our code to our different environments. This is a bundle of commits essentially. When viewing the environment shown in github it only shows the latest commit and not a list of commits. Is there anyway to have it collect all the commits between deployments?  A downside of this is that we cannot see in Jira all the items in their respective environment. Aside from the latest commit on the release branch.

**Comment by ossanna16** [comment]
Hi there @newtewt and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
